modules/web-console-overview.adoc

```diff
@@ -0,0 +1,40 @@
+// Module included in the following assemblies:
+// * assembly/architecture
+
+[id='web-console-overview-{context}']
+= Understanding the web console
+
+The web console runs as a pod on the master. The static assets required to run
+the web console are served by the pod. Administrators can also customize the web
+console using extensions, which let you run scripts and load custom stylesheets
+when the web console loads.
+
+When you access the web console from a browser, it first loads all required
+static assets. It then makes requests to the {product-title} APIs using the
+values defined from the `openshift start` option `--public-master`, or from the
+related parameter `masterPublicURL` in the `webconsole-config` config map
+defined in the `openshift-web-console` namespace. The web console uses
+WebSockets to maintain a persistent connection with the API server and receive
+updated information as soon as it is available.
+
+The configured host names and IP addresses for the web console are whitelisted
+to access the API server safely even when the browser would consider the
+requests to be cross-origin. To access the API server from a web application
+using a different host name, you must whitelist that host name by specifying the
+`--cors-allowed-origins` option on `openshift start` or from the related master
```

Comment:
@ahardin-rh do you know if `openshift start` is in 4.0?

Reply:
@bmcelvee Good question. I can get a cluster up tomorrow and check.

Reply:
@bmcelvee I'm not seeing it, so it could be replaced by `oc cluster up` or similar.